### PR TITLE
Add section to set up timing expectations ⌚️

### PR DESCRIPTION
**Pull request by fvcproductions**
Related to #155 @hiimbex - let me know if this fits the bill! 😊🤖️🕔

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -72,6 +72,14 @@ limitPerRun: 30
 #     - confirmed
 ```
 
+## How long will it take?
+
+The app runs on a scheduled basis and in batches in order to avoid hitting rate limit ceilings. 
+
+Ergo, even after you initially install the GitHub configuration and add the `stale.yml` file, you may not see it act immediately.
```
Comment (hiimbex) [inline README.md]: nit: 'ergo' isn't very commonly used in English, and I'd love for this to be as easily readable as possible. Instead maybe:

> What this means it that, even after...

**Comment by fvcproductions** [comment]
@hiimbex done! 👍

**Comment by hiimbex** [comment]
Sorry one more thing, could you move the section to be either just before or after ' ## Why did only some issues and pull requests get marked stale?' I think it's in a similar vein, so they belong near each other.

**Comment by fvcproductions** [comment]
@hiimbex Does that move work?

**Comment by hiimbex** [comment]
Perf! Thanks @fvcproductions!

**Pull request merged**